<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGame</class>
 <widget class="QDialog" name="CreateGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>360</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crear Partida</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNumPlayers">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>380</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cantidad de jugadores</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numPlayers">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="gameName">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>24</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreate">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>475</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>475</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>0</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
